test/controllers/api/v1/locations_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class Api::V1::LocationsControllerTest < ActionDispatch::IntegrationTest
+  test "POST /api/v1/code_schools/:code_school_id/locations" do
+    school = create(:code_school)
+    params = {
+      location: {
+        va_accepted: Faker::Boolean.boolean,
+        address1: Faker::Address.street_address,
+        address2: Faker::Address.street_address,
+        city: Faker::Address.city,
+        state: Faker::Address.state,
+        zip: Faker::Address.zip_code
+      }
+    }
+
+    post api_v1_code_school_locations_path(school, params)
+    assert_response :success
+  end
+
+  test "PATCH /api/v1/code_schools/:code_school_id/locations/:id" do
```

Comment:
Small nit.  Can you please separate out the **setup**, **action**, and **expectation** groupings of each test with a space.  i.e.

```ruby
test ":update PATCH /api/v1/code_schools/:code_school_id/locations/:id" do
  school = create(:code_school)
  location = create(:location)
  params = {
    location: {
      va_accepted: Faker::Boolean.boolean,
      address1: Faker::Address.street_address,
      address2: Faker::Address.street_address,
      city: Faker::Address.city,
      state: Faker::Address.state,
      zip: Faker::Address.zip_code
    }
  }

  patch api_v1_code_school_location_path(school, location, params: params)
  
  assert_response :success
end
```

Reply:
@hpjaj I would typically do this but the directory of quality assurance at my company insists I don't whenever I submit code reviews at work. It consistently sites "isolated." I don't like ignoring DRY either but I was just trying to bring what he tells me are "best practices" from work here. 

You're certain this is a non issue in these tests? Isolated that is.

Reply:
Oh sorry I just glanced over your comment, my bad. Ok let me do that.